src/plugins/textEmoji/styles.css

```
@@ -0,0 +1,3 @@
+.reaction_fef95b .emoji {
```

```suggestion
[class^="reaction_fef95b"] .emoji {
```
AA here we go again (I should've just looked at the whole thing in one go)

---

Is this really a good idea? After some discussion on [Discord](https://discord.com/channels/1015060230222131221/1134844326933954622/1188452136648978432) I am convinced it's not, and it'd be better to just update the plugin when it breaks. I reverted the change, though if the maintainers will want this I will re-introduce it.

---

partial matching is faster and more compatible. I don't understand why you wouldn't use it.

---

kode's suggestion is correct. do not hardcode class hashes

---

Re-introduced the change, as it is preferred by the project.